app/src/main/java/com/example/mediadb/data/model/dataresponse/ListMovieData.kt

```diff
@@ -7,7 +7,7 @@ data class ListMovieData(
     @SerializedName("page")
     val page: Int,
     @SerializedName("results")
-    val results: List<Result>,
+    val movies: MutableList<Movie>,
```

Comment:
Trừ id ra các trường này anh để nullable hết. Sở dĩ như vậy vì api hoàn toàn có thể trả về một giá trị null. Còn id sau này a phải dùng Room nữa nên không thể để nullable được sau a dùng tới Room sẽ thấy